src/transport/raw/BLE.cpp

```diff
@@ -45,15 +45,19 @@ void BLEBase::ClearState()
 {
     if (mBleLayer)
     {
+#if CONFIG_NETWORK_LAYER_BLE
```

Comment:
Why are we ending up in the BLE transport if CONFIG_NETWORK_LAYER_BLE is not set?

@kpschoedel